nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -103,7 +103,9 @@ public void initialize(String name, Map<String, String> options) {
       api = nessieClientBuilder.build(NessieApiV2.class);
     } else {
       throw new IllegalArgumentException(
-          "Unsupported client-api-version: " + apiVersion + ". Can only be v1 or v2");
+          String.format(
+              "Unsupported %s: " + apiVersion + ". Can only be v1 or v2",
```

Comment:
this uses now a mix of String.format and string concatenation. Would be good to only use String.format

Reply:
I think it would be good to add a test for this